docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
-## Deprecation Warnings
+### Incremental Updates
+
+In Katello 2.1 we provided a first look at incremental updates via the API. With the 2.2 release, incremental updates can be applied via the API, UI and CLI. Incremental updates are applying errata, packages or puppet modules to a content view which produces a minor version of that content view version. The updated version can then be pushed out to all the environments that the major version exists in, thereby making small updates available to any hosts that are using that particular major version of the content view. Users can find this functionality by visiting the errata index page and using either the bulk actions or viewing the details of an individual errata and clicking on the 'Content Hosts' tab. 
```

Comment:
The UI doesn't support packages or puppet modules yet and this makes it sound like it does.  Perhaps we just remove "packages or puppet modules" for now?